intel-retail/performance-tools
# Results format

## PR Checklist



## What are you changing?

Update igt output format to use the latest 1.28 version of intel-gpu-tools with csv output support.

## Issue this PR will close

close: #523

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- clone performance tools and retail-use-cases
- make build-benchmark-docker
- build pipeline (cd  ../../retail-use-cases/use-cases/dlstreamer/docker-compose_dlstreamer.yml && make build)
- cd benchmark-scripts
- python3 benchmark.py --retail_use_case_root ../../retail-use-cases --compose_file ../../retail-use-cases/use-cases/dlstreamer/docker-compose_dlstreamer.yml
- wait for completion
- make consolidate ROOT_DIRECTORY=results
- check results directory for summary.csv

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/performance-tools )
